.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Please use a [javascript gitignore file](https://gist.github.com/andreasonny83/b24e38b7772a3ea362d8e8d238d5a7bc) here. Remove any files that match this (like package-lock.json).

Reply:
The .gitignore file has been populated to filter more unwanted files, please review and let me know if anything needs to be added to it.